LICENSE.txt

```diff
@@ -3,7 +3,7 @@ LICENSE AND COPYRIGHT
 
 pgbouncer_fdw is released under the PostgreSQL License, a liberal Open Source license, similar to the BSD or MIT licenses.
 
-Copyright (c) 2021 Crunchy Data Solutions, Inc.
+Copyright (c) 2021-2022 Crunchy Data Solutions, Inc.
```

Comment:
all rights reserved?

Reply:
`©` instead of `(c)`? (At least that's what we've done in paar...)

Reply:
```suggestion
Copyright © 2021-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
```